eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -252,6 +252,9 @@ public function locationMoved($locationId, $oldParentId, $newParentId)
                 'newParent' => $newParentId,
             ]
         );
+        if ($oldParentId === $newParentId) {
+            return $newParentId;
+        }
```

Comment:
Don't think this logic belongs in cache, only side effect here is that cache gets cleared anyway.

Reply:
agree, is it okay if I'll remove this part here and do this instead?
(few lines below):
```
if ($oldParentId === $newParentId) {
    $this->cache->invalidateTags(['urlAlias-location-' . $locationId, 'urlAlias-location-path-' . $locationId]);
}
```
Or, may I just remove it? It will not bring any issues, as it's not something you would do often)

Reply:
what you suggest is ok, the risk is if the `$oldParentId !== $newParentId` logic changes but if so this code should be adapted too.

Reply:
done